integration/kubernetes/cleanup_env.sh

```diff
@@ -41,6 +41,7 @@ main () {
 	sudo systemctl stop "${CRI_RUNTIME}"
 
 	sudo kill -9 $(pidof "containerd-nydus-grpc") || true
+	sudo rm -rf "/var/lib/containerd-nydus" || true
```

Comment:
@ryansavino, as part of the cleanup, I'd also add:
```
for i in `sudo ctr -n k8s.io snapshot --snapshotter nydus list | grep -v KEY | cut -d' ' -f1`; do sudo ctr -n k8s.io snapshot --snapshotter nydus rm $i; done
```

Reply:
Thanks! I have added this.

Reply:
Sure thing. Also, I moved the nydus snapshot cleanup to right before the CRI_RUNTIME stop call. In case this might cause any issues.